src/backend/BackendInterface.cpp

```diff
@@ -20,16 +20,17 @@
 #include <ripple/protocol/Indexes.h>
 #include <ripple/protocol/STLedgerEntry.h>
 #include <backend/BackendInterface.h>
-#include <log/Logger.h>
+#include <util/log/Logger.h>
 
 using namespace clio;
 
 // local to compilation unit loggers
 namespace {
-clio::Logger gLog{"Backend"};
+util::Logger gLog{"Backend"};
 }  // namespace
 
-namespace Backend {
+namespace clio::backend {
```

Comment:
instead of backend, how do u think we call it clio::data?
backend is kinda large scope

Reply:
Would you also rename `backend` directory to `data`? currently the namespaces match their respective source directories. 
I think long term `data` is a good name - we will likely introduce some abstraction/layer for accessing data on top of what is now known as `BackendInterface`.. eventually.

What about `subscription`? Any better name ideas there? :)

Reply:
Ended up renaming `backend` to `data` and `subscriptions` to `feed`. 
Also renamed the directories same way.